serverlessworkflow/modules/ROOT/pages/data-index/data-index-service.adoc

```diff
@@ -35,6 +35,34 @@ There are several ways to deploy the {data_index_ref} service. But there are som
 . Provide the database connection properties, to allow data index store the indexed data. {data_index_ref} service does not initialize its database schema automatically. To initialize the database schema, you need to enable Flyway migration by setting QUARKUS_FLYWAY_MIGRATE_AT_START=true.
 . Define the `KOGITO_DATA_INDEX_QUARKUS_PROFILE` to set the way that the events will be connected (by default: `kafka-event-support`).
 
+[NOTE]
+====
+It is needed to prepare the {product_name} workflow to support the full communication with an external {data_index_ref} service.
+For this purpose, it is important to make sure the following addons are included:
+
+.{product_name} workflow addon dependencies to support the connection with external {data_index_ref}
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-events-process</artifactId> <1>
+</dependency>
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-process-management</artifactId> <2>
+</dependency>
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-source-files</artifactId> <3>
+</dependency>
```

Comment:
```suggestion

```

We can remove this, as it's needed only for the management console. Sorry, I know that I'm adding for my JIRA, but it's unrelated.